include/glow/Backends/CompilationOptions.h

```diff
@@ -33,6 +35,30 @@ struct CompilationOptions {
 
   /// Insert TraceEvents between all instructions for profiling.
   bool autoInstrument{false};
+
+  // Different precision modes. Note that convertToFP16 and quantize can be
+  // specified simultaneously.
+  bool convertToFP16{false};
+  bool quantize{false};
+  bool profileForQuantization{false};
```

Comment:
quantize and profileForQuantization should be mutually exclusive right

Reply:
Correct -- I suppose I could replace them with an enum for QuantizationMode. I wanted to still allow fp16 conversion + quantization.